### DISCUSSION TITLE
Easy way to lock a tab (other than pinning)

### DISCUSSION THREAD

**Discussion by mdunker** [tabwrangler/tabwrangler#447] [General]
I just discovered Tab Wrangler. I am a long time tab abuser, and over time, Chrome eventually makes me pay. Tab Wrangler seems like a perfect idea. I want Tab Wrangler to aggressively close the tabs I don't lock.

The problem happens when I encounter a site I want to look at later. I want to lock it. So far, I have two solutions: pin the tab (which is not part of my current workflow) or go into the Tab Wrangler menu, select Tab Lock, and then look through a list of tabs that cannot be alphabetically sorted or searched to find the tab I am on and click the box.

I think a search bar for the lock list would be a great addition. Even better, I'd like to use a keyboard shortcut to lock the current tab, or click the Tab Wrangler menu and then click a button that says "Lock current tab."

Maybe this isn't a feature other people want, but it would make Tab Wrangler indispensable for me.

Thanks for the awesome extension!

**Comment by morguldir** [comment]
isn't this right click menu what you want? Not sure how new it is 
![image](https://github.com/tabwrangler/tabwrangler/assets/21371374/7b574070-ca07-4eb2-bffd-a158da66471e)

**Comment by ssorallen** [comment]
> a keyboard shortcut to lock the current tab

The keyboard shortcut is now available in Chrome as well. Go to `Window > Extensions` and then click the `Keyboard shortcuts` tab to customize shortcuts for Tab Wrangler.
